frontend/src/components/InputBox.jsx

```diff
@@ -0,0 +1,14 @@
+const InputBox = ({ label, placeholder, id }) => {
+  return (
+    <>
+      <label className="font-semibold ml-2">{`${label}`}</label>
+      <textarea
+        className="bg-white text-base m-2 h-8 rounded-sm p-1 resize-none shadow-xl/10 shadow-slate-900"
+        id={`${id}`}
+        placeholder={`${placeholder}`}
```

Comment:
Why not just `id={id}`, `placeholder={placeholder}` ? They're already strings, right?